test/test_uri_parser.rb

```diff
@@ -0,0 +1,13 @@
+load 'uri_parser.rb'
+
+class TestParseUri < Test::Unit::TestCase
+  def test_parse_uri_local
+    assert_equal([["dh=500", "da=l", "ds=s"], "/images/example.jpg"],
+                 parse_uri("/local/small_light(dh=500,da=l,ds=s)/images/example.jpg"))
+  end
+  def test_parse_uri
+    assert_equal([["dh=500", "da=l", "ds=s"], "/images/example.jpg"],
+                 parse_uri("/small_light(dh=500,da=l,ds=s)/images/example.jpg"))
+  end
+end
+
```

Comment:
Fixed in https://github.com/wantedly/nginx-image-server/pull/44/commits/6af385f5e4a0b9b914fe04644ff66f5b0e5f4f65.